ui.apps/src/main/content/jcr_root/apps/etoolbox-link-inspector/clientlibs/link-inspector-ui/js/console-ui.filter.js

```diff
@@ -82,12 +83,21 @@
         $('<p>').html('By text').appendTo(dialog.content);
         $linkSubstringField.appendTo(dialog.content);
 
+
+        const $dialogActionsButtons = $('<div class="filter-dialog-action-buttons">');
+        const $resetBtn = $('<button data-dialog-reset is="coral-button" variant="default">').text(RESET_FILTER_LABEL);
         const $cancelBtn = $('<button is="coral-button" variant="default" coral-close>').text(CANCEL_LABEL);
         const $updateBtn =
             $('<button data-dialog-action is="coral-button" variant="primary" coral-close>').text(SUBMIT_FILTER_LABEL);
 
-        $cancelBtn.appendTo(dialog.footer);
-        $updateBtn.appendTo(dialog.footer);
+        $cancelBtn.appendTo($dialogActionsButtons);
+        $updateBtn.appendTo($dialogActionsButtons);
+
+        const $buttonGroup = $('<div class="filter-dialog-button-group">');
```

Comment:
Why do you need a special button group? It seems to be that you could just add the two buttons to the footer itself as usual

Reply:
Since the reset button was added 
<img width="354" alt="Screenshot 2024-08-19 at 13 03 29" src="https://github.com/user-attachments/assets/af56e688-63c9-4243-8613-17ae7be000bd">